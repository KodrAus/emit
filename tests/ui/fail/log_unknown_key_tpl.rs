extern crate emit;

fn main() {
    emit::info!("something went wrong: {x}");
}
