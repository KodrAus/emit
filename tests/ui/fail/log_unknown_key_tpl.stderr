error[E0425]: cannot find value `x` in this scope
 --> fail/log_unknown_key_tpl.rs:4:41
  |
4 |     emit::info!("something went wrong: {x}");
  |                                         ^ not found in this scope
